src/lib/helpers/queryKeys.ts

```diff
@@ -37,4 +37,13 @@ export const queryKeys = createQueryKeyStore({
       compounderType: string
     }) => [chainId, compounderType],
   },
+  holdings: {
+    list: ({
+      chainId,
+      user,
+    }: {
+      chainId: number
+      user: Address | undefined
+    }) => ["holdings", chainId, user ?? "0x"],
```

Comment:
You don't need to manually include `"holdings"` in the query key. It is automatically included from the parent.

Currently, this query key will be `["holdings", "list", "holdings", chainId, "0x"]`, when really you want it to be `["holdings", "list", chainId, "0x"]`

Reply:
didn't know before, noted